packages/app/src/debug/DebugContainer.vue

```diff
@@ -6,8 +6,13 @@
       </NoInternetConnection>
       <DebugLoading v-else-if="!userProjectStatusStore.hasInitiallyLoaded || userProjectStatusStore.project.isProjectConnected && isLoading" />
       <DebugError
-        v-else-if="showError"
```

Comment:
Is the `showError` prop used anymore? If not it should be removed from the `defineProps`